IpmiFeaturePkg/IpmiPowerRestorePolicy/IpmiPowerRestorePolicy.c

```diff
@@ -10,7 +10,10 @@
 #include <PiPei.h>
 #include <Library/DebugLib.h>
 #include <Library/IpmiCommandLib.h>
-#include <Library/PlatformPowerRestorePolicyConfigurationLib.h>
+#include <Guid/PlatformPowerRestorePolicy.h>
+#include <Library/PolicyLib.h>
+
+#define  POWER_RESTORE_POLICY_UNKNOWN  0x03     // "Get Chassis Status"

```

Comment:
Does this definition really belong here?

Reply:
This macro is related to value return by "Get Chassis status" IPMI Command. I could not find any other appropriate header file to put this macro.

Reply:
Okay, it is fine to sit here as is. I am just wondering if other components might need to use this as well.

Reply:
If other component would also need it in future, then we can create new header file to keep it.